doc/uftrace-live.md

```diff
@@ -39,21 +39,21 @@ COMMON OPTIONS
     See *TRIGGERS*.
 
 -D *DEPTH*, \--depth=*DEPTH*
-:   Set global trace limit in nesting level.
+:   Set global trace limit in nesting level.  See *FILTERS*.
 
 -t *TIME*, \--time-filter=*TIME*
-:   Do not show small functions under the time threshold.  If some functions
-    explicitly have 'trace' trigger, those are always traced regardless of
-    execution time.
+:   Do not show functions which run under the time threshold.  If some functions
+    explicitly have the 'trace' trigger applied, those are always traced
+    regardless of execution time.  See *FILTERS*.
 
 \--no-libcall
 :   Do not record library function invocations.  Library calls are normally
-    traced by hooking the dynamic linker's resolve function in the PLT.  One can
-    disable it with this option.
+    traced by hooking the dynamic linker's resolve function in the PLT.
+    One can disable it with this option.
 
 \--no-event
 :   Disable event recording which is used by default.  Note that explicit event
-    tracing by \--event option is not affected by this.
+    tracing by `--event` option is not affected by this.
```

Comment:
I think it's better to use "\\" for escape rather than enclosing with "\`"

Reply:
I will take a look later for many other places.